tools/bin/commands/minishift

```diff
@@ -1,4 +1,12 @@
 #!/bin/bash
+export PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
```

Comment:
Also would move this to top-level `syndesis` so that its common for all commands.